docs/developer.md

```diff
@@ -1,15 +1,18 @@
-# Contributing Guide
+# Development Workflow
 
-To contribute openebs/maya firstly, you have to fork the openebs/maya repository. But before that you need to create an issue or pick an existing issue from [open issues](https://github.com/openebs/openebs/issues)
+## Prerequisites
 
-In any case, before you start working on your issue, sync your repository with the upstream openebs/maya master. Syncing ensures your repository has the latest changes.
+* You have Go 1.9 installed on your host/development machine
+* You have Docker installed on your host/development machine. Docker is required for building maya components container images and to push them into Kubernetes Cluster for testing. 
 
-### 1. Fork in the cloud
+## Initial Setup
+
+### Fork in the cloud
 
 1. Visit https://github.com/openebs/maya
 2. Click `Fork` button (top right) to establish a cloud-based fork.
 
-### 2. Clone fork to local storage
+### Clone fork to local machine
 
 Place openebs/maya' code on your `GOPATH` using the following cloning procedure.
```

Comment:
following cloning command.

Reply:
it is more commands so leaving as procedure.